src/content/advanced/iam-roles-for-service-accounts/index.md

```diff
@@ -0,0 +1,105 @@
+---
+linkTitle: IAM Roles for Service Accounts
+title: AWS IAM Roles for Service Accounts 
+description: This article describes how to use a new feature that allows binding of specific AWS IAM roles to a service account of a pod.
+weight: 60
+menu:
+  main:
+    parent: advanced
+user_questions:
+ -  How can I use IAM Roles for Service Accounts?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2022-03-04
+---
+
+# IAM Roles for Service Accounts (IRSA)
+
+{{< platform_support_table aws="alpha=v17.1.0" >}}
+
+You can associate an IAM role with a Kubernetes service account. This service account can then provide AWS permissions to the containers in any pod that uses that service account. With this feature, you no longer need to provide extended permissions to the GiantSwarm node IAM role so that pods on that node can call AWS APIs.
+
+Applications must sign their AWS API requests with AWS credentials. This feature provides a strategy for managing credentials for your applications, similar to the way that Amazon EC2 instance profiles provide credentials to Amazon EC2 instances. Instead of creating and distributing your AWS credentials to the containers or using the Amazon EC2 instance’s role, you can associate an IAM role with a Kubernetes service account. The applications in the pod’s containers can then use an AWS SDK or the AWS CLI to make API requests to authorized AWS services.
+
+The official documentation from AWS: [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+
+## Enable the feature on your cluster
+This is an alpha feature that has to be enabled by setting an annotation on the  [`AWSCluster`]({{< relref "/ui-api/management-api/crd/awsclusters.infrastructure.giantswarm.io.md" >}}) resource.
+
+Make sure the resource has the `alpha.aws.giantswarm.io/iam-roles-for-service-accounts` annotation. The value can be anything you like, as only the presence of that annotation is checked. Here is an example:
+
+```yaml
+apiVersion: infrastructure.giantswarm.io/v1alpha3
+kind: AWSCluster
+metadata:
+  annotations:
+    alpha.aws.giantswarm.io/iam-roles-for-service-accounts: ""
+  labels:
+    aws-operator.giantswarm.io/version: 10.18.0
+    cluster.x-k8s.io/cluster-name: abcl0
+    giantswarm.io/cluster: abcl0
+    giantswarm.io/organization: myorg
+    release.giantswarm.io/version: 17.1.0
```

Comment:
If the labels are not relevant to the content here, we might as well omit them, so that they don't have to be maintained in the future. This way they also don't distract as much.

```suggestion
  labels:
    ...
```